### ISSUE TITLE
Feature request: Look for templates in subfolders inside blocks

### ISSUE THREAD

**Issue by landwire** [palmiak/timber-acf-wp-blocks#10]
Hi there,
I would like to use the following folder structure with the blocks:

```
blocks
    - myblock1
        - myblock1.twig
        - myblock1.config.php
        - myblock1.js
        - myblock1.scss
    - myblock2
        - myblock2.twig
        - myblock2.config.php
        - myblock2.js
        - myblock2.scss
    - etc.....
```

Unfortunately that does not work with the current code.
Could you extend your code to look into the subfolders? I could not get it to work yet.
I looked here for some ideas on how to do it:
https://stackoverflow.com/questions/17160696/php-glob-scan-in-subfolders-for-a-file

**Comment by palmiak** [comment]
For starters you can use the function you mentioned and use the `timber/acf-gutenberg-blocks-templates` filter - it's an array so you can add any path you like.

On the other hand I'll think about using subfolders - seeing your example makes sense to do so 👍

**Comment by landwire** [comment]
Just saw I never replied. Yes, that works fine now. I added the fix from Brent as well.

**Comment by palmiak** [comment]
Closed by #15
